.github/workflows/publish-charts.yaml

```diff
@@ -0,0 +1,90 @@
+name: publish-charts
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'Repository from where we want to build Charts'
+        required: true
+      repo_ref:
+        description: 'The branch/tag that will be used as new version for charts. Usually tag ref'
+        required: true
+      repo_chart:
+        description: 'Object with source path, destination path and chart name'
+        required: true
+
+env:
+  REPO: ${{ github.event.inputs.repo }}
+  REPO_REF: ${{ github.event.inputs.repo_ref }}
+
+jobs:
+  publish-charts:
+    runs-on: ubuntu-latest
+    name: Publish charts
+    continue-on-error: true
+    strategy:
+      max-parallel: 1
+      matrix: ${{ fromJson(github.event.inputs.repo_chart) }}
+    steps:
+
+      - id: checkout-gh-pages
+        uses: actions/checkout@v2 # Checkout Charts - gh-pages
+        with:
+          ref: 'gh-pages'
+          fetch-depth: 0
+
+      - id: checkout-helm-src
+        uses: actions/checkout@v2 # Checkout Helm source code
+        with:
+          repository: ${{ env.REPO }}
+          ref: ${{ env.REPO_REF }}
+          path: ${{ env.REPO }}
+
+      - name: Set chart name
+        id: chart-name
+        run: |
+          CHART=${{ matrix.chart }}-${{ env.REPO_REF }}.tgz
+          echo "::set-output name=name::$CHART"
+
+      - id: chart-version
+        name: 'Set version - ${{ env.REPO_REF }}'
+        if: ${{ steps.checkout-gh-pages.outcome == 'success' && steps.checkout-helm-src.outcome == 'success' }}
+        run: |
+          yq -i '.version = "${{ env.REPO_REF }}"' ${{ matrix.src_path }}/Chart.yaml
+          yq -i '.version = "${{ env.REPO_REF }}"' ${{ matrix.src_path }}/values.yaml
+          echo "Chart version: ${{ env.REPO_REF }}."
+        working-directory: ./${{ env.REPO }}
+
+      - id: check
+        name: 'Check if chart already exist'
+        if: ${{ steps.chart-version.outcome == 'success' }}
+        run: |
+          if [[ -f "${{ matrix.dst_path }}/${{ steps.chart-name.outputs.name }}" ]]; then 
+            echo "Chart already exists." 
+            echo "${{ matrix.dst_path }}/${{ steps.chart-name.outputs.name }}"
+            ls -larth ${{ matrix.dst_path }}
+            exit 1
+          fi
+
+          echo "File $FILE doesn't exist, continue with packaging"
```

Comment:
This variable is now missing.  it should be `${{ steps.chart-name.outputs.name }}`